crates/nu-cli/src/repl.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
     NuHighlighter, NuValidator, NushellPrompt,
 };
 use crossterm::cursor::CursorShape;
+use devtimer::SimpleTimer;
```

Comment:
I haven't used this before. Does it offer us useful features over something like this?

```rust
let start = Instant::now();
...
let end = Instant::now();
let duration: Duration = end.duration_since(start);
```

What's the motivation for using `devtimer` instead of

Reply:
@rgwood The problem I ran into with Instant is that there's no way to stop the timer. So, instead of having a timer run forever in our code, I chose to use a slim dependency that has a start and stop and other interesting features.

Reply:
Hmm, not sure I follow. In the example above with `std::time::{Instant, Duration}` there is no timer running forever; it's just calculating the difference between 2 Instants

Reply:
Maybe I was using it wrong. 

The code I used before was like 
```rust
let start_time = Instant::now();
```
then later in repl.rs
```rust
start_time.elapsed()
```
Then I noticed that if I kept running elapsed that it kept increasing, so I figure that meant that it was running. I can switch it back to Instant if you don't think that it's an issue.

Reply:
ok, will do. Thanks for your help!